go-controller/pkg/libovsdb/ops/portgroup.go

```diff
@@ -188,6 +188,31 @@ func AddACLsToPortGroupOps(nbClient libovsdbclient.Client, ops []libovsdb.Operat
 	return m.CreateOrUpdateOps(ops, opModel)
 }
 
+// UpdatePortGroupSetACLsOps updates the provided ACLs on the provided port group and
+// returns the corresponding ops
+// NOTE: If you pass a slice with 0 ACLs, then the acls will be wiped out
+func UpdatePortGroupSetACLsOps(nbClient libovsdbclient.Client, ops []libovsdb.Operation, name string, acls []*nbdb.ACL) ([]libovsdb.Operation, error) {
+	if acls == nil {
+		return ops, nil
+	}
```

Comment:
I think we shouldn't have different behavior with these two forms
```
UpdatePortGroupSetACLsOps(nbClient, ops, name, nil)
```
```
UpdatePortGroupSetACLsOps(nbClient, ops, name, []*nbdb.ACL{})
```

If we are weary of any ambiguity, then I think it would be better to ave separate explicit method to clear out the ACLs
```
UpdatePortGroupUnsetACLsOps(nbClient libovsdbclient.Client, ops []libovsdb.Operation, name string)
```

reminder to update the doc text as well if you change this

Reply:
Spoke to @jcaamano : I'll remove the special treatment we do for nil argument...

Reply:
done. also updated the doc text